docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -243,6 +243,8 @@ const classes = makeStyles(theme => ({
   +import AlertTitle from '@material-ui/core/AlertTitle';
   ```
 
+  You can use our [`moved-components` codemod](https://github.com/mui-org/material-ui/tree/HEAD/packages/material-ui-codemod#moved-components) for automatic migration.
```

Comment:
```suggestion
  You can use the [`moved-components` codemod](https://github.com/mui-org/material-ui/tree/HEAD/packages/material-ui-codemod#moved-components) for automatic migration.
```
(We generally try to avoid the use of possessive pronouns in the documentation).

Would "moved-lab-components" be more descriptive?

Reply:
> (We generally try to avoid the use of possessive pronouns in the documentation).

I'll try to keep it in mind, thanks!

> Would "moved-lab-components" be more descriptive?

Definitely!